content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx

```diff
@@ -114,25 +114,6 @@ Advanced `Json` filtering is supported by [PostgreSQL](/concepts/database-connec
 
 </Admonition>
 
-### Enable advanced filtering
-
-You can enable advanced filtering by editing the `previewFeatures` field in the `generator` block in your schema, then re-generating the Prisma Client:
```

Comment:
Should this maybe turned into a note when they were in preview? There are quite a few versions between 2.23.0 when this was introduced and now, so quite a few people will be using one of these version and might try to use the filters without the preview feature.

Reply:
Fair point - I'll add one.

Reply:
I've added this now